Chat_App/Core/Extensions/UIColor+Extensions.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  UIColor+Extensions.swift
+//  Chat_App
+//
+//  Created by Ana Dzamelashvili on 4/23/23.
+//
+
+import Foundation
+import UIKit
+
+extension UIColor {
+    convenience init(red: Int, green: Int, blue: Int) {
+        assert(red >= 0 && red <= 255, "Invalid red component")
```

Comment:
a rockstar badge for you if you came up with that by yourself. Defending your code from dumbasses is always a good practice